universum/modules/automation_server/github_server.py

```diff
@@ -0,0 +1,55 @@
+from ..error_state import HasErrorState
+from ...lib.module_arguments import ModuleArgumentParser
+from .base_server import BaseServerForHostingBuild, BaseServerForTrigger
+
+__all__ = [
+    "GithubServer",
+]
+
+
+class GithubServer(BaseServerForHostingBuild, BaseServerForTrigger, HasErrorState):
```

Comment:
You only need to implement BaseServerForTrigger if you are going to support triggering by this server from poller.